user_script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Read user information
+echo "Enter your first name:"
+read firstname
```

Comment:
All the variables in the script does not respect the guidelines of a variable in shell.